include/stl2/view/istream.hpp

```diff
@@ -19,21 +19,24 @@
 #include <stl2/detail/raw_ptr.hpp>
 #include <stl2/detail/semiregular_box.hpp>
 #include <stl2/detail/iostream/concepts.hpp>
+#include <stl2/detail/iterator/default_sentinel.hpp>
 #include <stl2/view/view_interface.hpp>
 
 STL2_OPEN_NAMESPACE {
 	namespace ext {
-		template<Semiregular Val>
+		template<Movable Val>
```

Comment:
Oops: We need to require `DefaultConstructible<Val>` so we can construct an object into which to extract a value. (Addressed in https://github.com/cjdb/cmcstl2/pull/24.)